tests/unit_tests/test_league_rater.py

```diff
@@ -0,0 +1,363 @@
+    assert new_score.score == current_score.score + config.SCORE_GAIN + config.POSITIVE_BOOST
+
+
+def test_new_score_defeat_no_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (180.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN
+
+    player_rating = (1800.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN
+
+
+def test_new_score_defeat_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (60.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN - config.NEGATIVE_BOOST
+
+
+def test_new_score_victory_highest_division_no_boost(example_league):
+    current_score = LeagueScore(division_id=3, score=5, game_count=30)
+    player_rating = (240.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN
+
+
+def test_new_score_victory_highest_division_boost(example_league):
+    current_score = LeagueScore(division_id=3, score=5, game_count=30)
+    player_rating = (380.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN + config.HIGHEST_DIVISION_BOOST
+
+
+def test_placement_after_enough_games(example_league, unplaced_player_score):
+    # Neutralize the offset in the placement function so we can test the score independently of the config settings
+    rating = (150.0 - config.RATING_MODIFIER_FOR_PLACEMENT, 0.0)
+
+    new_score = LeagueRater.rate(example_league, unplaced_player_score, None, rating)
```

Comment:
I am confused by the `None` in place of the game outcome? If you simply want the game outcome to have no effect on the score, maybe set it to draw? Then it adheres to the interface and doesn't get flagged in e.g. mypy type checks - although the logic is the same and python won't call you out on it.
```suggestion
    new_score = LeagueRater.rate(example_league, unplaced_player_score, GameOutcome.DRAW, rating)
```